lib/devise/strategies/yolo_authenticatable.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "devise/strategies/authenticatable"
+
+module Devise
+  module Strategies
+    class YoloAuthenticatable < Authenticatable
+      def authenticate!
+        if params[:user].present?
+          user = User.find_by(email: params[:user][:email])
+
```

Comment:
Can you add a ticket to JIRA to sort our logging admins in a better way?

Reply:
done https://dfedigital.atlassian.net/browse/CPDEL-258